appstudio-controller/controllers/appstudio.redhat.com/snapshotenvironmentbinding_controller.go

```diff
@@ -412,6 +413,14 @@ func generateExpectedGitOpsDeployment(component appstudioshared.BindingComponent
 		}
 	}
 
+	// Append labels to identify the Application, Component and Environment associated with this GitOpsDeployment.
+	// Label values are limited to a maximum of 63 characters, while resource names by default can be up to 253.
+	// StoneSoup artificially limits these names to 63 characters, but to be safe we truncate the values if they
+	// are too long
+	res.ObjectMeta.Labels[appstudioLabelKey+"/application"] = limitLength(binding.Spec.Application, logger)
```

Comment:
I suppose this is the right label key, and not the one that ends with .com as mentioned in the Jira?

Reply:
Yes, after a slack discussion with Jonathan, we decided it made more sense to use `appstudio.openshift.io` as the label.  I'll put a comment in the jira.

Reply:
Ok.